### PR TITLE
handle authenticator in http requests

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: octobot_tentacles_manager/api/util/tentacles_management.py [hunk 1]
```diff
@@ -19,14 +19,17 @@
...
 
 
-async def manage_tentacles(worker, tentacle_names, tentacles_path_or_url=None, aiohttp_session=None) -> int:
+async def manage_tentacles(worker, tentacle_names, tentacles_path_or_url=None, aiohttp_session=None,
+                           authenticator=None) -> int:
     errors_count = 0
     logger = logging.get_logger(__name__)
     try:
         if tentacles_path_or_url is not None:
+            session = authenticator.get_aiohttp_session() if authenticator and authenticator.is_logged_in() \
```
Comment (Herklos) [inline octobot_tentacles_manager/api/util/tentacles_management.py:28]: :+1:

Comment (Herklos) [reply octobot_tentacles_manager/api/util/tentacles_management.py:28]: Where is this Authenticator class defined ? In Commons ?

Comment (GuillaumeDSM) [reply octobot_tentacles_manager/api/util/tentacles_management.py:28]: here https://github.com/Drakkar-Software/OctoBot/blob/master/octobot/community/authentication.py

Comment (Herklos) [reply octobot_tentacles_manager/api/util/tentacles_management.py:28]: Ok

Comment (Herklos) [reply octobot_tentacles_manager/api/util/tentacles_management.py:28]: I don't like to manipulate object defined in the caller. I would expect to have this Authenticator defined in Commons. But if it's only for this call it's probably overkill to move this class declaration.

Comment (GuillaumeDSM) [reply octobot_tentacles_manager/api/util/tentacles_management.py:28]: It's strictly related to the octobot community features so to me it makes sense to have it in octobot, we could define an abstract class in octobot-commons though

Comment (Herklos) [reply octobot_tentacles_manager/api/util/tentacles_management.py:28]: :+1:

**Review by Herklos — APPROVED** [review]

**Pull request merged**